frontend/views/containers/payments/RecordPayment.vue

```diff
@@ -131,16 +131,16 @@ export default ({
     ]),
     paymentsList () {
       return this.todoItems.map(item => {
-        return item.status === PAYMENT_NOT_RECEIVED // if not received item, re-format the obj
```

Comment:
Turns out it has always been resolved as `false` here, even when it's actually in the `PAYMENT_NOT_RECEIVED'`status. So made a correction to this logic while working on the task.